components/infra-proxy-service/server/policyfiles.go

```diff
@@ -187,3 +185,37 @@ func (s *Server) DeletePolicyfile(ctx context.Context, req *request.DeletePolicy
 		Name: req.GetName(),
 	}, nil
 }
+
+// GetPolicyfileRevisions gets a policy file revisions
+func (s *Server) GetPolicyfileRevisions(ctx context.Context, req *request.PolicyfileRevisions) (*response.PolicyfileRevisions, error) {
+	c, err := s.createClient(ctx, req.OrgId, req.ServerId)
+	if err != nil {
+		return nil, err
+	}
+
+	policyfileRevision, err := c.client.Policies.Get(req.Name)
+	if err != nil {
+		return nil, ParseAPIError(err)
+	}
+
+	return &response.PolicyfileRevisions{
+		Revisions: fromAPIIncludedPolicyfileRevisions(policyfileRevision),
+	}, nil
+}
+
+// fromAPIIncludedPolicyfileRevisions a response included policyfile revision
+func fromAPIIncludedPolicyfileRevisions(p chef.PolicyGetResponse) []*response.PolicyfileRevision {
```

Comment:
We should add some unit tests for this considering that the response from server is going to change.